zerver/views/user_settings.py

```diff
@@ -140,7 +140,6 @@ def update_display_settings_backend(request, user_profile,
 
     return json_success(result)
 
-@authenticated_json_post_view
```

Comment:
Probably not needed. See https://github.com/zulip/zulip/pull/2822#issuecomment-268554315